realm/realm-library/src/main/java/io/realm/internal/Table.java

```diff
@@ -593,48 +569,6 @@ private boolean isPrimaryKeyColumn(long columnIndex) {
     }
 
     /**
-     * Returns a view sorted by the specified column and order.
-     *
-     * @param columnIndex the column index.
-     * @param sortOrder the sort order.
-     * @return a sorted view.
-     */
-    public TableView getSortedView(long columnIndex, Sort sortOrder){
```

Comment:
Nope, user should always build `TableView` from `Query`.  Sure for some cases build `TableView` directly from the `Table` will be `faster`, but I don't think it will be fast enough for worthy doing that -- that means we also need to handle this special case for asyc query especially.